src/main/java/picard/vcf/GenotypeConcordance.java

```diff
@@ -685,7 +685,11 @@ static String spliceOrAppendString(final String destination, final String toInse
             return destination.substring(0, insertIdx) + toInsert + destination.substring(insertIdx);
         }
         else {
-            return destination + toInsert;
+            if (destination.length() > 0 && destination.charAt(destination.length()-1) == '*') {
```

Comment:
Could you add a comment here stating what this does?

Reply:
Also, instead of '*', use Allele.SPAN_DEL_STRING

Reply:
@skwalker I think you missed this comment.

Reply:
oops yep, sorry!